shared/src/desktopMain/kotlin/org/icerock/template/main.desktop.kt

```diff
@@ -3,10 +3,11 @@ import androidx.compose.runtime.Composable
 
 actual fun getPlatformName(): String = "Desktop"
 
-@Composable fun MainView() = App()
+@Composable
+fun MainView() = App()
```

Comment:
rollback all changes of this file please

Reply:
done